src/sprites/rendered-target.js

```diff
@@ -767,6 +775,8 @@ RenderedTarget.prototype.toJSON = function () {
         direction: this.direction,
         draggable: this.draggable,
         costume: this.getCurrentCostume(),
```

Comment:
Now that we have the full costume list, perhaps this should update to the current costume id rather than the costume data?  (In a subsequent PR)